geoh5py/io/h5_writer.py

```diff
@@ -620,7 +638,7 @@ def write_data_values(  # pylint: disable=too-many-branches
                     name_map,
                     data=out_values,
                     compression="gzip",
-                    compression_opts=9,
```

Comment:
@MatthieuCMira Can you remind me why we specified compression to 9? Remember you asking me for the images...